integreat_cms/cms/migrations/0078_alter_poi_category.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.20 on 2023-07-25 15:46
+
+from django.db import migrations, models
+
+from ..constants import poicategory
+
+
+# pylint: disable=unused-argument
+def check_and_update_poicategory(apps, schema_editor):
+    """
+    This function checks whether the default POI category "Other" exists and cretes it if not.
+    It checks furthermore whether each location has a category and assigns the default category if not.
+    """
+    POICategory = apps.get_model("cms", "POICategory")
+    POI = apps.get_model("cms", "POI")
+
+    default_category, _ = POICategory.objects.get_or_create(
+        icon=poicategory.OTHER, defaults={"color": poicategory.DARK_BLUE},
+    )
```

Comment:
```suggestion
    default_category, created = POICategory.objects.get_or_create(
        icon=poicategory.OTHER,
        defaults={"color": poicategory.DARK_BLUE},
    )
    if created:
        POICategoryTranslation = apps.get_model("cms", "POICategoryTranslation")
        POICategoryTranslation.objects.create(
            category=default_category, language=default_language, name=poicategory.OTHER
        )

```
Maybe like this to create the translation?